iChef-WebApi/src/main/resources/application.properties

```diff
@@ -1,14 +1,13 @@
 spring.jpa.hibernate.ddl-auto=update
-spring.datasource.url=jdbc:mysql://localhost:3306/schema
+spring.datasource.url=jdbc:mysql://localhost:3306/theChefs
```

Comment:
@turcudaria @madalinazele @roxanatrufelea @Stefpark77 Can you notify me when you change this config file in a PR? If you change this file I also need to change something on the pipeline. Close this comment when either one of you sees this and tell the others about my request

Reply:
The config file won't be changed by this commit, we will let it be schema,root and 1234 for every push